app/helpers/mountain_view/styleguide_helper.rb

```diff
@@ -0,0 +1,11 @@
+module MountainView
+  module StyleguideHelper
+    def mv_components
+      Dir.glob(Rails.root.join('app', 'components', '*')).map { |component_dir| File.basename component_dir }
```

Comment:
Line is too long. [109/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.